src/utils/getPreviousValue.ts

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useRef } from 'react';
 
-export default function usePreviousValue(value: any): any {
+export default function usePreviousValue<T = any>(value: T): T {
```

Comment:
To be honest I don't think the `= any` part is needed for generics, as far as I know you can only extend interfaces etc here.